### ISSUE TITLE
airodump-ng won't save wps and uptime columns while exporting to csv file

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1724]
**Reported by mitroko on 9 Jul 2017 22:08 UTC**

Using airodump-ng 1.2 it is impossible to export network list with uptime and wps columns included.
Right now as a workaround I'm forced to use screenscrape to automate my tools, based on airodump-ng which is bad.
Please fix

**Comment by glezo1** [comment]
IMHO this should be considered as a bug, not as an enhancement

**Comment by aircrack-ng** [comment]
@glezo1 Changing the CSV format would break the existing tools relying on it.

**Comment by glezo1** [comment]
Well, I would've expected relying tools to parse the CSV including header line, so they would still work if more columns are added.
Got a doubt: will it (wps info column) be added to other output format, such as netxml?

**Comment by aircrack-ng** [comment]
No, netxml is pretty much fixed. This ticket is about creating a new format, most likely JSON so it doesn't break tools when extending it.

**Comment by AmirrezaNasiri** [comment]
An idea for migrating this compatibility issue is to use flags. For example:
```
Options:
  ...
  --output-flags       : Modify output files for different kinds of options. Flags separated by a comma:
                         wps,uptime,manufacturer 
```
Since these flags are set explicitly, it will work as the user intended.
